commcare_connect/opportunity/tasks.py

```
@@ -309,8 +309,18 @@ def send_push_notification_task(user_ids: list[int], title: str, body: str):
+RETRYABLE_EXCS = (httpx.ReadTimeout, httpx.ConnectTimeout)
+
+
+@celery_app.task(
+    bind=True,
+    autoretry_for=RETRYABLE_EXCS,
+    retry_kwargs={"max_retries": 5},
+    retry_backoff=True,
+    retry_backoff_max=300,
+    retry_jitter=True,
+)
+def download_user_visit_attachments(self, user_visit_id: int):
```

Ruff (0.14.11)

Unused function argument: `self`

(ARG001)